### ISSUE TITLE
Hyrule warriors - Asserts before ingame

### ISSUE THREAD

**Issue by JohnGodgames** [citra-emu/citra#2198]
Tested on https://github.com/citra-emu/citra/commit/2de470c9b26f709859b2d842e60a2303784b3cb0

**while assert in log:**
```
[  31.756412] HW.Memory <Error> core\memory.cpp:Memory::PhysicalToVirtualAddress:699: Unknown physical address @ 0x28685DB0
[  31.756456] HW.Memory <Error> core\memory.cpp:Memory::GetPointer:282: unknown GetPointer @ 0xa8685db0
```

![hyrule](https://cloud.githubusercontent.com/assets/11453507/20463070/ae1efa48-af2b-11e6-9a80-23ef5d90394d.PNG)

**Stack trace:**
```
>	citra-qt.exe!Pica::CommandProcessor::WritePicaReg(unsigned int id, unsigned int value, unsigned int mask) Line 249	C++
 	citra-qt.exe!Pica::CommandProcessor::ProcessCommandList(const unsigned int * list, unsigned int size) Line 469	C++
 	citra-qt.exe!GPU::Write<unsigned int>(unsigned int addr, const unsigned int data) Line 489	C++
 	citra-qt.exe!GSP_GPU::ExecuteCommand(const GSP_GPU::Command & command, unsigned int thread_id) Line 502	C++
 	citra-qt.exe!GSP_GPU::TriggerCmdReqQueue(Service::Interface * self) Line 625	C++
 	citra-qt.exe!Service::Interface::SyncRequest() Line 85	C++
 	citra-qt.exe!SVC::SendSyncRequest(unsigned int handle) Line 238	C++
 	citra-qt.exe!HLE::Wrap<&SVC::SendSyncRequest>() Line 149	C++
 	citra-qt.exe!SVC::CallSVC(unsigned int immediate) Line 1151	C++
 	[External Code]	
 	citra-qt.exe!Dynarmic::BackendX64::EmitX64::GetBasicBlock(Dynarmic::IR::LocationDescriptor descriptor) Line 53	C++
```

**Comment by mailwl** [comment]
@JohnGodgames may you try other variants, without Dynarmic, or disable Shader JIT?

**Comment by JohnGodgames** [comment]
@mailwl tried with all off HW, Shader JIT, CPU JIT, Scaled resolution.....the same

**Comment by Subv** [comment]
This might be a bug uncovered by #2195

**Comment by benderscruffy** [comment]
still the same problem in the latest nightly and canary
[log.zip](https://github.com/citra-emu/citra/files/1502809/log.zip)
![hyrule](https://user-images.githubusercontent.com/2696404/33227273-82e6117c-d1da-11e7-839b-ef2d9be52dbb.PNG)

**Comment by FearlessTobi** [comment]
Fixed by #4392.
